python/cugraph/utilities/utils.py

```diff
@@ -21,8 +21,25 @@
     from cupyx.scipy.sparse.coo import coo_matrix as cp_coo_matrix
```

Comment:
I could use the new `import_optional()` utility defined below, but that would mean I'd either have to define it above the imports, or import all optional dependencies at the end of the file after the function is defined.